aws/resource_aws_batch_compute_environment.go

```diff
@@ -33,6 +33,15 @@ func resourceAwsBatchComputeEnvironment() *schema.Resource {
 				MaxItems: 1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"allocation_strategy": {
```

Comment:
Can you please add a covering acceptance test for this new argument in `aws/resource_aws_batch_compute_environment_test.go`? Copy-paste-modify of an existing test configuration and function is 👍 . Thank you.

Reply:
many thanks for the review, I have done both of these steps hopefully good to go